pkg/controller/keycloak/keycloak_migrations.go

```diff
@@ -64,3 +81,57 @@ func findDeployment(desiredState common.DesiredClusterState) *v13.StatefulSet {
 	}
 	return nil
 }
+
+func oneTimeLocalBackup(cr *v1alpha1.Keycloak, desiredState common.DesiredClusterState) (common.DesiredClusterState, error) {
+	backupJob := findBackupJob(desiredState)
+	switch {
+	case backupJob == nil:
```

Comment:
Once we switch to creating `KeycloakBackup` CR and waiting till it's done, this whole part can go away.